main.rb

```diff
@@ -0,0 +1,8 @@
+require './app'
+
+def main
```

Comment:
I am unable to run the app using any of the options given. There is an error that is shown on every option. Kindly see the error below
![error2](https://user-images.githubusercontent.com/35504149/170672769-023f48c9-79db-4e98-a1aa-a719558c81f0.png)
.

Reply:
Hi @SajjadAhmad14 
okay, you right i did fix